clang-tools-extra/clang-tidy/add_new_check.py

```diff
@@ -511,7 +511,7 @@ def has_auto_fix(check_name: str) -> str:
 
         return ""
 
-    def process_doc(doc_file: Tuple[str, str]) -> Tuple[str, Optional[re.Match[str]]]:
+    def process_doc(doc_file):
```

Comment:
Change the `re.Match[str]` to `typing.Any`? Plus maybe a `FIXME Python 3.9` comment?